src/reducers/externalLibraries.ts

```diff
@@ -3,67 +3,57 @@ import {
   ExternalLibraryNames
 } from '../components/assessment/assessmentShape';
 
-/**
- * Defines all the external symbols for playground, i.e full access to runes functionality.
- */
-const TwoDRunesExternals = [
-  'show',
-  'color',
-  'random_color',
-  'red',
-  'pink',
-  'purple',
-  'indigo',
-  'blue',
-  'green',
-  'yellow',
-  'orange',
-  'brown',
-  'black',
-  'white',
-  'scale_independent',
-  'scale',
-  'translate',
-  'rotate',
-  'stack_frac',
-  'stack',
-  'stackn',
-  'quarter_turn_right',
-  'quarter_turn_left',
-  'turn_upside_down',
-  'beside_frac',
-  'beside',
-  'flip_vert',
-  'flip_horiz',
-  'make_cross',
-  'repeat_pattern',
-  'black_bb',
-  'blank_bb',
-  'rcross_bb',
-  'sail_bb',
-  'corner_bb',
-  'nova_bb',
-  'circle_bb',
-  'heart_bb',
-  'pentagram_bb',
-  'ribbon_bb'
-];
-
 /**
  * Defines which external libraries are available for usage, and what
  * external symbols (exposed functions) are under them.
  */
+
 const libEntries: Array<[ExternalLibraryName, string[]]> = [
   [ExternalLibraryNames.NONE, []],
-  [ExternalLibraryNames.TWO_DIM_RUNES, TwoDRunesExternals],
   [
-    ExternalLibraryNames.THREE_DIM_RUNES,
+    ExternalLibraryNames.RUNES,
     [
-      ...TwoDRunesExternals,
-      'anaglyph',
+      'show',
+      'color',
+      'random_color',
+      'red',
+      'pink',
+      'purple',
+      'indigo',
+      'blue',
+      'green',
+      'yellow',
+      'orange',
+      'brown',
+      'black',
+      'white',
+      'scale_independent',
+      'scale',
+      'translate',
+      'rotate',
+      'stack_frac',
+      'stack',
+      'stackn',
+      'quarter_turn_right',
+      'quarter_turn_left',
+      'turn_upside_down',
+      'beside_frac',
+      'beside',
+      'flip_vert',
+      'flip_horiz',
+      'make_cross',
+      'repeat_pattern',
+      'square',
+      'blank',
+      'rcross',
+      'sail',
+      'corner',
+      'nova',
+      'circle',
+      'heart',
+      'pentagram',
+      'ribbon',
       'hollusion',
-      'animate',
```

Comment:
Are `animate`, `stereogram` etc not being used anymore?

Reply:
animate: that's only a local function in the hollusion function. That function has been broken for more than a year. Someone probably put animate in the externalLibraries as a desperate attempt to fix hollusion. The problem lies elsewhere. Will try to fix this tomorrow.

I did fix a few more things since you reviewed it, Daryl.

The libraries are now cleaned up and tested. Good to merge, I'd say.